src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -2233,6 +2235,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <Output TaskParameter="FilesWritten" ItemName="FileWrites"/>
       <Output TaskParameter="DependsOnSystemRuntime" PropertyName="DependsOnSystemRuntime"/>
       <Output TaskParameter="DependsOnNETStandard" PropertyName="_DependsOnNETStandard"/>
+      <Output TaskParameter="UnresolvedAssemblyConflicts" ItemName="UnresolvedAssemblyConflicts"/>
```

Comment:
We might want to use a name that's more likely to be unique.  Similar to the pattern for the input property, that could be `ResolveAssemblyReferenceUnresolvedAssemblyConflicts`.